packages/cli-node/report.api.md

```diff
@@ -112,6 +116,14 @@ export type LockfileDiffEntry = {
   range: string;
 };
 
+// @public
+export const packageFeatureType: readonly [
+  '@backstage/BackendFeature',
+  '@backstage/BackstagePlugin',
```

Comment:
do we really need those one? it's for the old frontend system right?

Reply:
I'm not sure what needs to go here, it's what ever the types are for all of the frontend/backend plugins and modules. We need to keep in mind to capture any old stuff from where packages may not have been migrated yet?